src/ui/public/agg_table/__tests__/_group.js

```diff
@@ -24,13 +24,35 @@ import fixtures from 'fixtures/fake_hierarchical_data';
 import { LegacyResponseHandlerProvider } from '../../vis/response_handlers/legacy';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
 import { VisProvider } from '../../vis';
+import { tabifyAggResponse } from '../../agg_response/tabify';
```

Comment:
can we use also here absolute paths `from 'ui/agg_response/tabify`?

Reply:
nope, should use absolute paths inside the same module